README.md

```diff
@@ -167,6 +167,10 @@ steps:
 
 > Note: The `id` defined in `actions/cache` must match the `id` in the `if` statement (i.e. `steps.[ID].outputs.cache-hit`)
 
+## Known limitation
+
+- The `action/cache` is not supported on GithHub Enterprise Server deployments. GitHub Enterprise Server uses self-hosted runners, dependencies are typically cached on the runner by whatever dependency management tool is being used (npm, maven, etc.), so saving these to an external cache would be less efficient.
```

Comment:
```suggestion
- `action/cache` is currently not supported on GitHub Enterprise Server. <https://github.com/github/roadmap/issues/273> is tracking this.

Since GitHub Enterprise Server uses self-hosted runners, dependencies are typically cached on the runner by whatever dependency management tool is being used (npm, maven, etc.), which eliminates the need for caching in some scenarios.
```

The issue is that caching needs new functionality added to work on prem.  It needs to use user-provided storage in the same way that logs, artifacts, and packages do, rather than the cloud storage it uses on github.com.